cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/network/compare/Comparison.java

```diff
@@ -448,7 +448,7 @@ private void compareTwoWindingTransformers(TwoWindingsTransformer expected,
 
         compareRatioTapChanger(expected.getRatioTapChanger(), actual.getRatioTapChanger());
         comparePhaseTapChanger(expected.getPhaseTapChanger(), actual.getPhaseTapChanger());
-        comparePhaseAngleClock(expected.getExtension(TwoWindingsTransformerPhaseAngleClock.class), actual.getExtension(TwoWindingsTransformerPhaseAngleClock.class));
+        comparePhaseAngleClock(expected.<TwoWindingsTransformerPhaseAngleClock>getExtension(TwoWindingsTransformerPhaseAngleClock.class), actual.<TwoWindingsTransformerPhaseAngleClock>getExtension(TwoWindingsTransformerPhaseAngleClock.class));
```

Comment:
Without this we have a build error with java 8 (ambiguous reference)

Reply:
Okay ! Rather than that, could you then rename the methods `comparePhaseAngleClock` respectively as `comparePhaseAngleClock2` and `comparePhaseAngleClock3`? It would be more readable.